src/net_processing.cpp

```diff
@@ -6216,27 +6284,6 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                     // No reason to drain out at many times the network's capacity,
                     // especially since we have many peers and some will draw much shorter delays.
                     unsigned int nRelayedTransactions = 0;
-
-                    size_t inbounds_fanout_tx_relay = 0, outbounds_fanout_tx_relay = 0;
```

Comment:
Do you mean squashing these changes instead of having a "move" commit?

Reply:
yes

Reply:
Because it is doing more than just moving the code around, so I thought it may be easier for reviewers already familiar with the original approach to understand the changes. I wouldn't mind squashing it once all the discussion around `RelayTranaction` is resolved

Reply:
My impression was that there are very-very few `reviewers already familiar with the original approach` at the moment, so it's an extra burden for most to understand that former code you drop anyway.
But yeah it's ok. Perhaps a commit message saying it will be dropped will work.

Reply:
I'll squash in the next force push